packages/quantic/cypress/e2e/default-1/sort/sort.cypress.ts

```diff
@@ -104,6 +154,53 @@ describe('quantic-sort', () => {
           });
       });
 
+      describe('when passing custom options as slots to the component', () => {
+        it('should display the custom options properly', () => {
+          visitSort({useCase: param.useCase}, true, {
+            withCustomSortOptions: true,
+          });
+
+          Expect.displaySortDropdown(true);
+          Expect.labelContains('Sort by');
+
+          Actions.openDropdown();
+
+          Expect.containsOptions(customSortOptionsValues);
+          Expect.optionsEqual(customSortOptions);
+        });
+
+        customSortOptionsValues
+          .filter((value) => value !== defaultCustomOptionValue)
+          .forEach((value) => {
+            describe('when selecting a custom option', () => {
+              it(`should update the shown selected option to ${value}`, () => {
+                visitSort({useCase: param.useCase}, true, {
+                  withCustomSortOptions: true,
+                });
+                Actions.selectOption(value);
+
+                Expect.selectedOption(value);
+                Expect.completeSearchRequest(
+                  'resultsSort',
+                  param.useCase,
+                  (body) => Expect.sortCriteriaInSearchRequest(body, value)
+                );
+                Expect.logSortResults(value);
+              });
+            });
+          });
+
+        describe('when the custom option passed has an invalid property', () => {
+          it('should display an error message instead of the quanticSort component', () => {
+            visitSort({useCase: param.useCase}, true, {
+              withInvalidCustomSortOptions: true,
+            });
+
+            Expect.displayComponentError(true);
```

Comment:
We should also use `Epect.displayComponentErrorMessage` to make sure the error was displayed correctly.